scheduler/scheduler/config/configs.go

```diff
@@ -1,74 +1,84 @@
 package config
 
+import (
+	"time"
+
+	"github.com/twitter/scoot/scheduler/server"
+	"github.com/twitter/scoot/workerapi/client"
+)
+
 // SchedulerConfigs the map of available configurations
-var SchedulerConfigs = map[string]string{
+var ServiceConfigs = map[string]ServiceConfig{
 	"default":      defaultConfig,
```

Comment:
could use const strings to define these keys now, looks like a few random bits of code use these magic strings